ftl/core/fields.ftl

```diff
@@ -12,3 +12,5 @@ fields-reverse-text-direction-rtl = Reverse text direction (RTL)
 fields-size = Size:
 fields-sort-by-this-field-in-the = Sort by this field in the browser
 fields-that-field-name-is-already-used = That field name is already used.
+field-name-first-letter-not-valid = The field name should not start with #, ^ nor /.
+field-name-invalid-letter = The field name should not contain :, { nor }.
```

Comment:
Note for next time - the prefix should match the filename, even if it's not grammatical - eg fields-first-letter-not-valid